src/com/company/onCrease.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
+
+public class onCrease {
```

Comment:
class names should have first letter of each word in caps and objects should follow camel case notation

Reply:
Noted. Will take care in future. Meanwhile removed this as was no longer needed.